MailCore/Utils/SentryDebug.swift

```diff
@@ -149,6 +149,16 @@ public enum SentryDebug {
         return crumb
     }
 
+    private static func addAsyncBreadcrumb(level: SentryLevel,
+                                           category: String,
+                                           message: String,
+                                           data: [String: Any]? = nil) {
+        Task.detached {
```

Comment:
ok but here as `SentryDebug` is not `mainActor` you probably do not need the `detached`

Reply:
Absolutely correct, will fix 👍